io.github.ferraridamiano.ConverterNOW.yaml

```diff
@@ -0,0 +1,45 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/master/data/flatpak-manifest.schema.json
+app-id: io.github.ferraridamiano.ConverterNOW
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: converternow
+separate-locales: false
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --share=ipc
+  - --device=dri
+modules:
+  - name: converternow
+    buildsystem: simple
+    build-commands:
+      - cp -r ConverterNOW /app/
+      - chmod +x /app/ConverterNOW/converternow
+      - mkdir -p /app/bin
+      - ln -s /app/ConverterNOW/converternow /app/bin/converternow
```

Comment:
so basically you install x86_64 binaries but will let flatpak generate an aarch64 version.

if you can't build from source (preferred) then you should follow this:
https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64

Reply:
Added `flathub.json`

Reply:
:-(